src/main/java/com/google/sps/servlets/ScheduledInterviewServlet.java

```diff
@@ -95,4 +110,62 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       return;
     }
   }
+
+  public List<ScheduledInterviewRequest> objectToRequestObjects(
+      List<ScheduledInterview> scheduledInterviews, String timeZoneIdString) {
+    ZoneId timeZoneId;
+    if (timeZoneIdString == null) {
+      timeZoneId = ZoneId.systemDefault();
+    } else {
+      timeZoneId = ZoneId.of(timeZoneIdString);
+    }
+    List<ScheduledInterviewRequest> requestObjects = new ArrayList<ScheduledInterviewRequest>();
+    for (ScheduledInterview scheduledInterview : scheduledInterviews) {
+      requestObjects.add(makeSheduledInterviewRequest(scheduledInterview, timeZoneId));
+    }
+    return requestObjects;
+  }
+
+  private String getDateString(TimeRange when, ZoneId timeZoneId) {
+    LocalDateTime start = LocalDateTime.ofInstant(when.start(), timeZoneId);
+    LocalDateTime end = LocalDateTime.ofInstant(when.end(), timeZoneId);
+    String startTime = start.format(DateTimeFormatter.ofPattern("h:mm a"));
+    String endTime = end.format(DateTimeFormatter.ofPattern("h:mm a"));
+    String day = start.format(DateTimeFormatter.ofPattern("EEEE, MMMM d, yyyy"));
+    return day + " from " + startTime + " to " + endTime;
```

Comment:
Use String.format() here instead of String concatenation.

Reply:
Done